Wrappers/Python/ccpi/viewer/utils/colormaps.py

```diff
@@ -465,6 +465,61 @@ def get_color_transfer_function(cls, cmap, color_range):
 
         return tf
 
+    @staticmethod
+    def get_lookup_table(cmap, color_range):
```

Comment:
is this meant to be here?
What exactly is it doing?

Reply:
I don't think it should be here.

Reply:
at any rate I fixed a bug in `get_lookup_table`

Reply:
get_lookup_table didn't previously exist - I don't understand what is it for?

Reply:
removed